### DISCUSSION TITLE
Why not use rojopolis/spellcheck-github-actions for spelling checks?

### DISCUSSION THREAD

**Discussion by andy5995** [curl/curl#10560] [Q&A] [answered]
Is there a particular reason you're using [UnicornGlobal/spellcheck-github-actions](https://github.com/UnicornGlobal/spellcheck-github-actions) for the spellcheck workflow?

It's a fork of [rojopolis ](https://github.com/rojopolis)/
[spellcheck-github-actions](https://github.com/rojopolis/spellcheck-github-actions). The UnicornGlobal fork is 14 commits ahead/264 commit behind the original. There haven't been any new commits to the fork since March of 2020. The original was updated 4 days ago.

**Comment by bagder** [comment, marked as answer]
I have no good answer to that. That was the spellchecker I found and it worked.
